web/templates/twig-source/incuna-surveys/form/base/proportion-field.html

```diff
@@ -1,16 +1,34 @@
-<div class="proportion">
-    {% block title %}<h4 ng-bind="title"></h4>{% endblock %}
-    {% block total %}Total: <span class="total" ng-bind="total"></span>{% endblock %}
-    {% block fields %}
-    <div ng-repeat="field in fields">
-        {% block field %}
-        <div drf-form-field="field" class="proportion">
-            {% block input %}
-                <span integer-field model="model[$index]" id="field.id" form="form"></span>
+{% block title %}<h4 class="title" ng-bind="title"></h4>{% endblock %}
+{% block total-bar %}
+    <span class="total-bar">
+        <span class="bar-portion {% block bar-portion-class %}{% endblock %}" ng-repeat="field in fields" style="width: {% raw %}{{ field.percentage }}%{% endraw %}"></span>
+    </span>
+{% endblock %}
+{% block total %}
+    <span class="total">
+        Total: <span class="total-figure" ng-bind="total"></span>
+    </span>
+{% endblock %}
+{% block fields %}
+    <div class="fields-wrapper">
+        <div class="proportion-field" ng-repeat="field in fields">
+            {% block field %}
+                <div drf-form-field="field" class="proportion-field-inner {% block field-class %}{% endblock %}">
+                    {% block input %}
+                        <input
```

Comment:
If you need the `proportion-input` class then can it be on the span `<span integer-field class="proportion-input" model="model[$index]" id="field.id" form="form"></span>`